src/ui/app/App.js

```diff
@@ -344,11 +344,11 @@ const getRouteComponent = () => {
 };
 
 class App extends Component {
-  componentDidUpdate(prevProps) {
```

Comment:
I remember this was purposely changed to `componentDidUpdate` because there is some kind of error when fetching the `plugins` because of some auth or smth. Can you revert it back and write preferences fetching in `componentDidMount` ? Important is, that `fetchPlugins` is inside `componentDidUpdate`

Reply:
I checked with @gidesan, and we said it would be ok to move to `componentDidMount`. He might not be aware of that though, will keep the `fetchPlugins` on componentDidUpdate to avoid any issues.